.buildkite/hooks/pre-command

```diff
@@ -2,6 +2,11 @@
 set -euo pipefail
 [[ -n "${DEBUG:-}" ]] && set -x
 
+if [[ "${BUILDKITE_AGENT_META_DATA_PLATFORM}" == "macos" ]] ; then
```

Comment:
We can do the same thing we do in `post-checkout` instead of adding a new block?

Reply:
checked with filip, we will have the capable_of_building tag on the machines so we can simplify this